example/src/config.js

```diff
@@ -1,7 +1,7 @@
 import { PhylloEnvironment } from "react-native-phyllo-connect"
 
-const clientId = '<client id here>' // add your Id here
-const clientSecret = '<client secret here>' // add your client Secret
+const clientId = '0a62ced6-3b00-4112-9c6b-7e7e61db02d0' // add your Id here
```

Comment:
Always remove keys from commits. Add this step to the release process checklist.

Reply:
yes its in checklist . we are doing testing on this branch .

Reply:
Even for testing, either use ENV vars or use keys only in local build, don't commit in code.